presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -776,7 +776,7 @@ export class QueryDetail extends React.Component {
                     cpuTimeRate: addToHistory(currentCpuTimeRate, this.state.cpuTimeRate),
```

Comment:
can you explain in cmt message why this is better?

Reply:
user memry cna be 0 (eg when doing TS) while query still consumes memory (and can be killed due to that when limits are low)